nixos/modules/programs/sway.nix

```diff
@@ -139,7 +153,7 @@ in {
     fonts.enableDefaultFonts = mkDefault true;
     programs.dconf.enable = mkDefault true;
     # To make a Sway session available if a display manager like SDDM is enabled:
-    services.xserver.displayManager.sessionPackages = [ swayPackage ];
+    services.xserver.displayManager.sessionPackages = optional (cfg.package != null) [ cfg.package ];
```

Comment:
Same issue here. You can catch these by fiddling with `nixos/tests/sway.nix`

Reply:
Fixed sorry, run the tests and should be it now ✌🏻